packages/overlay/trigger-directive.md

```diff
@@ -36,26 +44,79 @@ type InsertionOptions = {
 };
 ```
 
-#### Consumption via `lit-html`
+## Examples
 
 Pass a `TemplateResult` into the `trigger()` directive, as follows in order to have it rendered to the DOM when the associated Overlay is about to open and the removed after the Overlay has closed.
```

Comment:
Can you also help the consumers understand the specific use cases where they will try to think through this implementation?